webapi/src/test/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala

```diff
@@ -751,10 +761,13 @@ class KnoraSipiIntegrationV1ITSpec
       // Upload the PDF file to Sipi.
       val pdfUploadResponse: SipiUploadResponse = uploadToSipi(
         loginToken = loginToken,
-        filesToUpload = Seq(FileToUpload(path = pathToMinimalPdf, mimeType = MediaTypes.`application/pdf`))
+        filesToUpload = Seq(
+          FileToUpload(path = pathToMinimalPdf, mimeType = org.apache.http.entity.ContentType.create("application/pdf"))
+        )
       )
 
       val uploadedPdfFile: SipiUploadResponseEntry = pdfUploadResponse.uploadedFiles.head
+      println(uploadedPdfFile)
```

Comment:
Do we need all these added println statements in this file or was it just for local testing?